_data/content.yml

```
@@ -47,3 +47,35 @@
     <a href="https://donate.doctorswithoutborders.org/onetime.cfm" target="_blank" rel="noopener"><img src="img/icons/msf.png" alt="@MSF logo"></a>
     <a href="https://www.nilc.org/get-involved/donate/" target="_blank" rel="noopener"><img src="img/icons/nilc.png" alt="@NILC logo"></a>
     </div>
+    
+- section: FAQ
+  id: faq
+  text: |
+  
+    - *Where is it? Where should I stay? What's that about Lyft credit?* 
+        
+        Details are in the paper invitation and email sent upon your RSVP.
+
+    - *What should I wear?* 
+        
+        Clothes! Dressy casual is recommended; extra points for happy colors.
+    
+    - *What time should we be there?* 
+        
+        We encourage you to come at 3PM Pacific Standard Time, and hope to start the ceremony at 3:30PM. 
```

Summertime is Pacific Daylight Time (PDT is probably just as clear and less wordy).
```suggestion
        We encourage you to come at 3:00 <small>PM</small> (PDT), and hope to start the ceremony at 3:30 <small>PM</small>. 
```